test/acceptance/README.md

```diff
@@ -1,125 +1,77 @@
 # Acceptance Tests
 
-> Future Farming and Countryside Programme - AHWR Apply Acceptance Tests
+> Future Farming and Countryside Programme - AHWR Dashboard Acceptance Tests
 
-This folder contains the acceptance tests for the FFC AHWR apply service components.
+This folder contains the acceptance tests for the FFC AHWR Dashboard service components.
 
-The framework is (Cucumber)[https://cucumber.io/] and (webdriver.io)[https://webdriver.io/] based, containerised, expandable and based on the actively maintained webdriver.io Cucumber boilerplate project.
+The framework is using (Cucumber)[https://cucumber.io/] and (webdriver.io)[https://webdriver.io/]
 
 ## Requirements
 
-- Docker Desktop 2.2.0.3 (42716) or higher
-- Node version 16
-  - if you have multiple versions of node, run `nvm use 16` to use version 16 for this alone
+- Docker 
+- Node
+- npm
 
-# Quick start
+## Test execution in CI build pipeline
+When the tests are run by Jenkins the `TEST_ENVIRONMENT_ROOT_URL` variable is generated and exposed by the PR build pipeline to point to the PR-specific Sandbox deployment.
 
-Docker is used to create a container for each of selenium-hub, chrome-browser and webdriver-cuke.
-* Selenium Hub allows concurrent execution of test cases
-* Chrome Browser is the browser specified in the configuration file `wdio.bs.config.js` by default
-* Webdriver.io along with Cucumber is this framework that defines the tests.
+## Test execution locally
 
-## How to run the tests
+### Environment Variables
+Provide the following environment variables, typically in a `.env` file in the `/test/acceptance` directory:
 
-1. Set the root URL for the environment in the environment variable `TEST_ENVIRONMENT_ROOT_URL`
-
-2. If running against localhost, then no need to set `TEST_ENVIRONMENT_ROOT_URL` as it will default to `docker.host.internal:3000`.  Instead make sure the application container is running with `docker-compose up --build` in the root folder of this repository
-
-3. Set the Browserstack credentials in the environment variables `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY`
+```pwsh
+TEST_ENVIRONMENT_ROOT_URL
+CRN_USERNAME
+CRN_PASSWORD
+```
 
-4. Set the number of Browserstack parallel runs in the environment variable `BROWSERSTACK_PARALLEL_RUNS`
+### Running tests inside a container
+Docker is used to create containers for both the tests (`wdio-cucumber`) and the Selenium instance of Chrome (`chrome-browser`). This is also how the tests run in the Jenkins pipeline. Headless browser mode is used.
 
-5. From the directory containing the dockerfile run `docker-compose run --rm wdio-cucumber`. This will run an acceptance test against the FFC AHWR apply service. Alternatively, run `./scripts/acceptance` from the root folder of this repository.
+1. For ARM architectures, change the image used for Chrome in `docker-compose.yaml`:
```

Comment:
We could actually add a switch for this to do it automatically, but not a drama for now